endoreg_db/models/medical/patient/patient_examination.py

```
@@ -144,6 +148,13 @@ def get_indication_choices(self) -> List["ExaminationIndicationClassificationCho
         ]
         return choices
 
+    def get_or_create_patient_examination_by_id(self, pk: int) -> Optional["PatientExamination"]:
+        """Hilfsmethode zum Abrufen oder Erstellen einer PatientExamination nach ID"""
+        if not self.objects.filter(pk=pk).exists():
+            return None
+        else:
+            return self.objects.filter(pk=pk)
```

Pylint (3.3.7)

[refactor] Unnecessary "else" after "return", remove the "else" and de-indent the code inside it

(R1705)